### PR TITLE
APPS-193: Fixed the issue 

### PR DESCRIPTION
**Pull request by bogdanioan-gabor**
Fixed the issue  by overriding the web.config bean and added the share-config.properties file as a property location.

Cleaned aims configuration:
 - Replaced "aims.config." prefix with "aims." prefix
 - Removed complexity of creating AIMSConfig (removed AIMSConfigElement and unused AIMSConfigElementReader classes)

### CODE REVIEW

File: share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java [hunk 1]
```diff
@@ -41,9 +40,8 @@
 
 public class AIMSConfigTest extends BaseTest
 {
-
-    public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
-    private static final String TEST_CONFIG_AIMS_BASIC_XML = "test-config-aims-basic.xml";
+    // public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
```
Comment (cristina-sirbu) [inline share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java:43]: Why this class has been commented out?

Comment (bogdanioan-gabor) [reply share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java]: @cristina-sirbu I've commented the AIMS tests, the AIMSConfig was changed considerably. I've added I minimal set of tests for the updated AIMSConfig.

File: share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java [hunk 2]
```diff
@@ -25,139 +25,109 @@
...
 public class AIMSConfigTest extends BaseTest
 {
-
-    public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
+    public static final String CLASSPATH_SHARE_CONFIG_PROPERTIES = "classpath*:alfresco/module/*/share-config.properties";
     private static final String TEST_CONFIG_AIMS_BASIC_XML = "test-config-aims-basic.xml";
-    protected XMLConfigService configService;
 
-    protected List<String> getConfigFiles()
+    @Before public void setUp()
```
Comment (cristina-sirbu) [inline share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java:46]: Why do we need this?

Comment (bogdanioan-gabor) [reply share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java]: @cristina-sirbu Removed the unneeded setUp method.

**Pull request merged**